e2e/server_fuzz_test.go

```diff
@@ -0,0 +1,57 @@
+package e2e_test
+
+import (
+	"fmt"
+	"testing"
+	"unicode"
+
+	"github.com/Layr-Labs/eigenda-proxy/client"
+	"github.com/Layr-Labs/eigenda-proxy/e2e"
+	op_plasma "github.com/ethereum-optimism/optimism/op-plasma"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func addAllUnicodeTestCases(f *testing.F) {
+	for r := rune(0); r <= unicode.MaxRune; r++ {
+		if unicode.IsPrint(r) {
+			f.Add(fmt.Sprintf("seed: %s", string(r)), []byte(string(r))) // Add each printable Unicode character as a seed
+		}
+	}
+}
+
+// FuzzProxyClientServerIntegrationAndOpClientKeccak256MalformedInputs will fuzz the proxy client server integration
+// and op client keccak256 with malformed inputs
+func FuzzProxyClientServerIntegrationAndOpClientKeccak256MalformedInputs(f *testing.F) {
+	if !runFuzzTests {
+		f.Skip("Skipping test as FUZZ env var not set")
+	}
+
+	testCfg := e2e.TestConfig(useMemory())
```

Comment:
is there any need to run this against testnet?

Reply:
Not in the test i had written. Will add more tests incrementally.